### ISSUE TITLE
[Bug]: FTB Ultimine not active while having extended player reach

### ISSUE THREAD

**Issue by Eric-Schnitte** [FTBTeam/FTB-Mods-Issues#1060]
### Mod

FTB Ultimine

### Mod version

ftb-ultimine-fabric-2001.1.2

### Forge / Fabric version

1.20.1 0.14.24

### Modpack & version

BigChadGuys Plus-1.1.7

### What issue are you having?

When using the Item "Arm Extender" from the mod "Things" (things-0.3.3+1.20) the player gets an additional 2 attack range and 2 reach. When i use this item i can breack blocks further away then possible in vanilla but i cant use ultimine. Ultimine is active as long as im in vanilla range but when i go back a block ultimine switches to not active and i cant break multiple blocks.
![2024-01-04_14 14 19](https://github.com/FTBTeam/FTB-Mods-Issues/assets/117738821/40c6729d-29a1-4253-bc88-7a33d957d0af)
![2024-01-04_14 14 24](https://github.com/FTBTeam/FTB-Mods-Issues/assets/117738821/5b40edfa-2da7-4c01-9317-5e38ea0e2c38)

### Steps to reproduce

1. Put Arm Extender in glove trinket slot
2. Activate Ultimine
3. Walk back untli it says not active
4. You should still be able to breack the block just not multiple

**Comment by desht** [comment]
Right, the reach distances is currently hardcoded to 5.0 on Fabric 1.20.1.  This is because Fabric itself (unlike Forge) doesn't provide a custom entity attribute for reach distance.  In 1.21, this is handled by vanilla so both NeoForge and Fabric have equivalance there.

Trinkets on Fabric 1.20.1 handles this via the https://github.com/JamiesWhiteShirt/reach-entity-attributes mod, which we should also be able to hook into for Ultimine on Fabric 1.20.1.

**Comment by desht** [comment]
Fixed in 2001.1.6 release
